graphdatascience/graph/graph_object.py

```diff
@@ -17,11 +16,10 @@ class Graph:
     It contains summary information about the graph.
     """
 
-    def __init__(self, name: str, query_runner: QueryRunner, server_version: ServerVersion):
+    def __init__(self, name: str, query_runner: QueryRunner):
         self._name = name
         self._query_runner = query_runner
         self._db = query_runner.database()
-        self._server_version = server_version
```

Comment:
thought we use the server_version to derive which yields to use in the graph_info 🤔

Reply:
(see https://neo4j.com/docs/graph-data-science/current/management-ops/graph-list/#tnoteref1)

Reply:
no tests are failing. do you know where the feature would be implemented?